website/project/model.py

```diff
@@ -1786,19 +1789,27 @@ def pointing_at(self, pointed_node_id):
                 return pointer._id
         return None
 
-    def get_points(self, folders=False, deleted=False, resolve=True):
-        ret = []
-        for each in self.pointed:
-            pointer_node = get_pointer_parent(each)
-            if not folders and pointer_node.is_collection:
-                continue
-            if not deleted and pointer_node.is_deleted:
-                continue
-            if resolve:
-                ret.append(pointer_node)
-            else:
-                ret.append(each)
-        return ret
+    def get_points(self, folders=False, deleted=False):
+        if not self.pointed:
+            return []
+
+        # Note: Due to GUIDs being unique there will never be an overlap between Nodes and pointers
+        pointers = {
+            x._backrefs['parent']['node']['nodes'][0]: x._id
```

Comment:
You will want to check this versus develop, and this may explain the other backref question above. https://github.com/CenterForOpenScience/osf.io/pull/5307 has cleared away almost all of the backrefs.